classes/class.ilOpencastPageComponentPluginGUI.php

```diff
@@ -133,166 +155,71 @@ public function performCommand(string $cmd): void
             case self::CMD_RESET_FILTER:
             case self::CMD_SHOW_UPLOAD_FORM:
             case self::CMD_UPLOAD:
+            case self::CMD_REPLACE:
                 $this->{$cmd}();
                 break;
             default:
                 break;
         }
     }
 
-    protected function addToolbar(): void
-    {
-        $upload_button = ilLinkButton::getInstance();
-        $upload_button->setPrimary(true);
-        //        $this->dic->ctrl()->saveParameter($this, 'rtoken'); // TODO why???
-        $this->dic->ctrl()->setParameter($this, self::CUSTOM_CMD, self::CMD_SHOW_UPLOAD_FORM);
-        $upload_button->setUrl($this->dic->ctrl()->getLinkTarget($this, self::CMD_INSERT));
-        $upload_button->setCaption($this->plugin->txt('btn_upload'), false);
-        $this->dic->toolbar()->addButtonInstance($upload_button);
-    }
+    // we want to reducet the code here, therefore we delegate some thing to other classes (TODO)
```

Comment:
Either remove this comment or write format the TODO properly:

```suggestion
    // TODO: we want to reducet the code here, therefore we delegate some thing to other classes.
```

Reply:
I will remove the line